catalog/ui/catalog-ui-search/src/main/webapp/component/query-status/query-status-header.hbs

```diff
@@ -13,25 +13,13 @@
  --}}
 <th>
     Source
-    <div class="column-absolute">
-    Source
-    </div>
 </th>
-<th>
+<th data-help="This is the number of results that made it into the Top.">
```

Comment:
Nit: "made it into the Top" seems less clear than just saying "This is the number of results returned"